### PR TITLE
Fix Terminus Update instructions

### PR DESCRIPTION
**Pull request by carolyn-shannon**
## Summary

**[Terminus Version Updates](https://pantheon.io/docs/terminus/updates)** - Fixes Terminus guide Version Updates instructions to separate instructions for standalone and Composer-managed Terminus versions. Adds guidance to [`self:update` command](https://pantheon.io/docs/terminus/commands/self-update) reference page.

## Post Launch

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6001](https://pr-6001--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `53` | `51` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6001` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23740/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23740/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6001--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/terminus/updates.md [hunk 1]
```diff
@@ -40,14 +51,18 @@ releases.
...
 
 ```bash{promptUser: user}
 rm -rf $HOME/terminus mkdir $HOME/terminus cd $HOME/terminus curl -O
 https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar
 php installer.phar install
 ```
 
+### Self:update not defined
+
+The `self:update` command is only available for standalone Terminus installed using the [standalone Terminus PHAR](/terminus/install#standalone-terminus-phar). If `self:update` returns a not defined error, use the [Terminus Installer PHAR](#update-terminus-installer-phar) update instructions above.
```
Comment (EdwardAngert) [inline source/content/terminus/updates.md:64]: @carolyn-shannon I saw that you added the partial file with this info - is it meant to take this paragraph's place?

**Review by EdwardAngert — CHANGES_REQUESTED** [review]

File: source/content/terminus/updates.md (hunk 1 above)
Comment (alexfornuto) [reply source/content/terminus/updates.md:64]: @EdwardAngert that partial file is under the `terminus` subdir and matches a terminus command, which means it's built into the command reference page for `terminus self:update`.

**Review by alexfornuto — APPROVED** [review]
LGTM, Nice command reference partial!

**Pull request merged**